### ISSUE TITLE
List of placeholder sprites

### ISSUE THREAD

**Issue by Colin-Tel** [DeltaV-Station/Delta-v-rebase#17]
From PR https://github.com/DeltaV-Station/Delta-v-rebase/pull/13
```
Textures/DeltaV/Objects/Weapons/Guns/Pistols/viperwood.rsi/bolt-open.png
Textures/DeltaV/Objects/Weapons/Guns/Revolvers/webleysnubnose.rsi/bolt-open.png
Textures/DeltaV/Objects/Weapons/Guns/Rifles/vulcan.rsi/bolt-open.png
Textures/DeltaV/Objects/Weapons/Guns/SMGs/typewriter.rsi/bolt-open.png
```

**Comment by TadJohnson00** [comment]
Should be fixed, fixed, fixed, and fixed as of PR #130 :3

**Comment by FluffiestFloof** [comment]
Was fixed in #154
